src/utils/agents/swap.ts

```diff
@@ -528,9 +528,11 @@ async function processAgentSwap(agent: AOYieldAgent, walletAddress: string): Pro
       const recentTxs = await getRecentTxs();
       await setRecentTxs([...recentTxs, { id: messageId, timestamp: Date.now(), queryCount: 0 }]);
 
-      // Schedule staggered checks for transaction success
-      [0.5, 1, 1.5, 2].forEach((delay) => {
-        browser.alarms.create(AO_YIELD_AGENT_RECENT_TXS_CHECK_ALARM_NAME, { when: Date.now() + delay * 60 * 1000 });
+      // Schedule alarm to check if the swap was successful
```

Comment:
Clearing and starting an alarm after a minute for every 2 minute regular interval check